common/lib/dependabot/clients/azure.rb

```diff
@@ -172,7 +172,7 @@ def create_commit(branch_name, base_commit, commit_message, files,
 
       # rubocop:disable Metrics/ParameterLists
       def create_pull_request(pr_name, source_branch, target_branch,
-                              pr_description, labels, work_item = nil)
+                              pr_description, labels, reviewers, work_item = nil)
```

Comment:
Note to self: We could make this an optional arg instead of required here... depends whether we make `reviewers` a global required arg vs a global optional or local to azure arg.

I want to noodle on this API a little...